### DISCUSSION TITLE
what is http.Handle("/auto.js.map", live.JavascriptMap{})?

### DISCUSSION THREAD

**Discussion by chrismalek** [jfyne/live#62] [Q&A] [answered]
I am trying to understand the concepts of this not having worked with too much front end javascript. This project looks epic.

I am trying to figure out what this is doing which I found in the examples.

`http.Handle("/auto.js.map", live.JavascriptMap{})`

**Comment by jfyne** [comment, marked as answer]
The map file is used by browser dev tools for debugging.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/SourceMap

It is unnecessary to include it for production, and is only good for tracking down frontend errors.
